Examples/Tests/pec/CMakeLists.txt

```diff
@@ -30,3 +30,13 @@ add_warpx_test(
     diags/diag1000020  # output
     OFF  # dependency
 )
+
+add_warpx_test(
+    test_2d_pec_field_insulator  # name
+    2  # dims
+    2  # nprocs
+    inputs_test_2d_pec_field_insulator  # inputs
+    analysis_default_regression.py  # analysis
+    diags/diag1000010  # output
+    OFF  # dependency
+)
```

Comment:
Given the complexity of the new feature, is there a physics analysis that can be done, beyond the checksums analysis?

Also, is this intentionally tested only in 2D? The implementation seems to be general for all dimensions, so maybe it could be worth adding tests for those as well. What do you think?

Reply:
This test was copied from the test case for the same boundary condition in the picnic code. That code's primary use is 2D. I don't know of any physics analysis that could be done beyond checking that the fields at the boundary are as expected. I could add other cases that do that simple check.